doc/reference/reports.rst

```diff
@@ -1,31 +1,101 @@
-.. _reference/reports:
-
 ============
 QWeb Reports
 ============
 
-Paper Formats
--------------
 
-There is a model called Paper Format allowing to define details specific to
-the PDF output.  These details include margins, header line, ... Everything
-related to the printed pdf.  Defining a paper format is not mandatory as there
-is a default one set on the company.  If you want a specific report to be
-associated to a specific paper format , just link the ir.actions.report.xml to
-it.
+Reports are written in HTML/QWeb, like all regular views in Odoo. You can use the usual QWeb control flow tools like t-if, t-foreach, etc. The PDF rendering is done by a tool outside the Odoo framework, **Wkhtmltopdf**.
+
+If you wish to create a report on a certain model, you will need to define this :ref:`report` and the :ref:`templates` it will use. If you wish, you can also specify a specific :ref:`paper_formats` for this report. Finally, if you wish to access more than your model, you can define a :ref:`custom_reports` class that gives you access to more models and records in the template.
+
+.. _report:
```

Comment:
References should be "namespaced" to avoid risks of collisions. rST doesn't have native reference namespaces but the `/` character is allowed so we're using that. See e.g. `data.rst` for examples.